packit_service/models.py

```diff
@@ -873,6 +885,13 @@ def get_by_pipeline_id(cls, pipeline_id: str) -> Optional["TFTTestRunModel"]:
                 .first()
             )
 
+    @classmethod
+    def get_all(cls, first: int, last: int) -> Optional[Iterable["TFTTestRunModel"]]:
```

Comment:
This is using `first` and `last` so it will never be really `all`, is it? :smile: 

How about `get_range()` or something similar?

Reply:
Sure